modules/oe_translation_cdt/src/Plugin/views/field/TargetLanguagesWithTooltip.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\oe_translation_cdt\Plugin\views\field;
+
+use Drupal\Component\Render\FormattableMarkup;
+use Drupal\Component\Render\MarkupInterface;
+use Drupal\Core\Language\LanguageManagerInterface;
+use Drupal\oe_translation_cdt\TranslationRequestCdtInterface;
+use Drupal\oe_translation_remote\TranslationRequestRemoteInterface;
+use Drupal\views\Plugin\views\field\FieldPluginBase;
+use Drupal\views\ResultRow;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Field handler to show the target languages with more information.
+ *
+ * @ingroup views_field_handlers
+ *
+ * @ViewsField("oe_translation_cdt_target_languages_with_tooltip")
+ */
+class TargetLanguagesWithTooltip extends FieldPluginBase {
+
+  /**
+   * Creates a new TargetLanguagesWithTooltip object.
+   *
+   * @param array $configuration
+   *   The configuration.
+   * @param string $plugin_id
+   *   The plugin ID.
+   * @param mixed $plugin_definition
+   *   The plugin definition.
+   * @param \Drupal\Core\Language\LanguageManagerInterface $languageManager
+   *   The language manager.
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, protected LanguageManagerInterface $languageManager) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition,
+      $container->get('language_manager')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function query(): void {
+    // Leave empty to avoid performing a query on this field.
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function render(ResultRow $values): string|MarkupInterface {
+    /** @var \Drupal\oe_translation_cdt\TranslationRequestCdtInterface $request */
+    $request = $this->getEntity($values);
+    $target_languages = $request->getTargetLanguages();
+    $total_count = count($target_languages);
+    $grouped = $this->groupLanguagesByStatus($target_languages);
+    $this->sortGroupedStatuses($grouped);
+
+    // Prepare the items for the tooltip.
+    $items = [];
+    foreach ($grouped as $status => $languages) {
+      $items[] = new FormattableMarkup('<strong>@status</strong>: @languages', [
+        '@status' => $status,
+        '@languages' => implode(', ', $languages),
+      ]);
+    }
+
+    // Count the number of translated languages.
+    $translated_count = 0;
+    foreach ($target_languages as $language_with_status) {
+      if (isset($request->getTranslatedData()[$language_with_status->getLangcode()])) {
+        $translated_count++;
+      }
+    }
+
+    $elements = [
+      'tooltip' => [
```

Comment:
Not sure why, but when trying locally it is not loading the css library so it doesn't actually display as a tooltip

Reply:
At first, I suspected that ePoetry pulls this library, but it is managed directly by `oe_translation`.

After few installs, the tooltip disappeared for me too :(

The oe_translation.module defines the [theme](https://github.com/openeuropa/oe_translation/blob/ba8f0504dde4b57acb129b3fecab4248f60c4c54/oe_translation.module#L39) and adds the tooltop library in the [preprocess function](https://github.com/openeuropa/oe_translation/blob/ba8f0504dde4b57acb129b3fecab4248f60c4c54/oe_translation.module#L348). It looks pretty straightforward. I am still checking it.

Reply:
It took me some time to track this, it turned out that the `renderPlain` method was dropping the asserts. I've changed it to `render` in https://github.com/openeuropa/oe_translation/pull/310/commits/90ac5ce8bfaede6341dfdb31c88d31b88695dae2 and then adjusted the test in https://github.com/openeuropa/oe_translation/pull/310/commits/5fa7dc383c471d038c9bd353ce7c4fdb15a485eb.